packaging/build_wheels.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -eux
+
+#	L I B R A R Y
+
+build_wheel() { (
+	# runs isolated in a sub-shell.
+
+	local runtime="$1"
+	local sdist="$2"
+
+	# Activate runtime in a subshell
+	PATH="$runtime/bin:$PATH"
+	hash -r
+
+	# Build wheel from source tarball
+	pip --verbose wheel --no-deps --wheel-dir "$distdir" "$sdist" 2>&1 | tee /tmp/wheel.log
+	wheelname="$(grep --max-count 1 --perl-regexp --only-matching "${tarname}.+\\.whl" /tmp/wheel.log)"
+
+	# Fix wheel to embed libev shared object.
+	auditwheel repair --wheel-dir "$distdir" "$distdir/$wheelname" 2>&1 | tee /tmp/wheel.log
```

Comment:
Should probably append to `wheel.log` using `tee -a`?

Reply:
Reusing the same file name may be misleading. The wheel is renamed. Thus, I need to get the filename twice for previous command output. I can update this to reduce log parsing.